populate/src/main/resources/seed/portals/hearthive/portal.json

```diff
@@ -56,7 +56,6 @@
       "password" : "broad_institute",
       "acceptingRegistration" : true,
       "initialized" : true,
-      "participantHostname" : "thehearthive.org",
```

Comment:
I don't think this should be here- I almost registered a user in hearthive prod because my local UI pointed me there 🙃